### ISSUE TITLE
Unable to launch OptiFine on 1.19.3 with Prism 6.3

### ISSUE THREAD

**Issue by CrazyCat0702** [PrismLauncher/PrismLauncher#862]
### Operating System

Ventura 13.0.1

### Version of Prism Launcher

6.3

### Version of Qt

6.3.0

### Description of bug

I'm attempting to launch OptiFine with 1.19.3 (I2 Pre3). I can launch with official launcher no problem. I've followed the instructions on your wiki page here: https://prismlauncher.org/wiki/getting-started/installing-optifine/

I've tried it both with and without moving the notch folder contents to the root. I just don't see any indication that OptiFine is enabled when launching. 

Thank you!

### Steps to reproduce

Follow instructions on Wiki for OptiFine installation, launch 1.19.3 from Prism.

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by byquanton** [comment]
Works without any issues ...
Did you run the Optifine Installer and used the Extracted Jar? 

![image](https://user-images.githubusercontent.com/32410361/219122071-70f03c04-ffd7-496a-96e1-4cc4c5bc30f2.png)

[Also have a look at alternatives ...](https://prismlauncher.org/wiki/getting-started/install-of-alternatives/)

**Comment by ww11xxx** [comment]
Im also having this issue i've tried it all the extracted one,the one from downloads and the one with the modified notch folder the optifine options don't show up in game i've gone to the reddit and discord but i could't find help or people with this same problem

**Comment by byquanton** [comment]
Can you a screenshot of the contents of your Optifine jar? It should look something like this:

![image](https://user-images.githubusercontent.com/32410361/219127285-6909a0a2-28dc-4952-a637-7fef3e854fac.png)

**Comment by ww11xxx** [comment]
![image](https://user-images.githubusercontent.com/53063288/219128994-c11ea107-35cf-4fd4-bc13-553853a070f8.png)
this is the one with the modified notch folder

**Comment by ww11xxx** [comment]
ok found the issue idk why it worked but i switched from 7zip to winrar and it is working now

**Comment by ghost** [comment]
You're supposed to run the installer and click on extract

![image](https://user-images.githubusercontent.com/110736028/219132547-6daf28d2-2b1c-4cbc-a2d5-e08d73a0b127.png)

**Comment by CrazyCat0702** [comment]
Ok, so similarly to @ww11xxx, it was an issue with my archiver. After moving the contents of the notch folder, I just used the macOS archive utility to create a .zip which I added to the Minecraft.jar. This did not work.

Instead, I used the following command to create an 'option_modded.jar' which I then added to the Minecraft.jar and now it works.
jar cfm ../opti_modded.jar META-INF/MANIFEST.MF *

Thanks for the hint, @ww11xxx!
